src/bin/pgcopydb/pgsql.c

```diff
@@ -2274,6 +2274,13 @@ pgsql_execute_log_error(PGSQL *pgsql,
 
 	LinesBuffer lbuf = { 0 };
 
+	if (message != NULL)
+	{
+		/* make sure message is writable by splitLines */
+		message = strdup(message);
+		/* Because we link with the libc Garbage Collector, we don't need to call free later */
```

Comment:
```suggestion

		/* Because we link with the libc Garbage Collector, we don't need to call free later */
```

Our indentation tool expects an empty space before a comment if it is followed by code. If we add this extra line here, the tests will pass and we will be able to merge the PR.

Reply:
I'd prefer it if we made it a multi-line comment really, in the style of:

``` c
    /*
     * Because we link ...
     * ...
     */
```

Reply:
done